Ali | Amir / Storybook for components (#2009)
binary-com/deriv-app: packages/components/stories/dialog/README.md

# Dialog Component

> Renders a `Dialog` component. The dialog may has Confirm and Cancel buttons.

Reviewer: ```suggestion
Renders a `Dialog` component. The dialog can have Confirm and Cancel buttons.
```
